_security/access-control/users-roles.md

```diff
@@ -75,6 +89,23 @@ See [YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#roles
 
 See [Create role]({{site.url}}{{site.baseurl}}/security/access-control/api/#create-role).
 
+## Edit Roles
+
+### OpenSearch Dashboards
+
+1. Choose **Security**, **Roles**, and under **Create role**, select **Explore existing roles**. 
+1. Given you are logged in with the correct admin permissions for editing the role, select the role from the list you would like to edit. 
```

Comment:
I would spin out the admin permissions note in this step as warning or note at the top of this section.

```suggestion
1. Select the role you would like to edit. 
```